src/ctelnet.cpp

```diff
@@ -1161,26 +1161,26 @@ void cTelnet::setGMCPVariables(const QString& msg)
     QString var;
     QString arg;
     if (msg.indexOf('\n') > -1) {
-        var = msg.section("\n", 0, 0);
-        arg = msg.section("\n", 1);
+        var = msg.section(QStringLiteral("\n"), 0, 0);
+        arg = msg.section(QStringLiteral("\n"), 1);
     } else {
-        var = msg.section(" ", 0, 0);
-        arg = msg.section(" ", 1);
+        var = msg.section(QStringLiteral(" "), 0, 0);
+        arg = msg.section(QStringLiteral(" "), 1);
     }
 
-    if (msg.startsWith("Client.GUI")) {
+    if (msg.startsWith(QStringLiteral("Client.GUI"))) {
         if (!mpHost->mAcceptServerGUI) {
             return;
         }
 
         QString version = msg.section('\n', 0);
-        version.replace("Client.GUI ", "");
-        version.replace('\n', " ");
+        version.replace(QStringLiteral("Client.GUI "), QString());
```

Comment:
`QString::remove(arg)` is so much cleaner than `QString::replace(arg, NULL)` and the following looks so much neater as:

    version.replace(QChar::LineFeed, QChar::Space)

Reply:
Fixed